script/validate_formatting

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Find incorrectly formatted files
+unformatted=$(gofmt -l .)
```

Comment:
We only had a few simple offences, so I opted to fix them and run for everything to keep things simpler. We could also change this to only run on differences.